sopel/tools/jobs.py

```diff
@@ -127,75 +199,284 @@ def _call(self, job):
 class Job(object):
     """Holds information about when a function should be called next.
 
-    :param int interval: number of seconds between calls to ``func``
-    :param func: function to be called
-    :type func: :term:`function`
+    :param intervals: set of intervals; each is a number of seconds between
+                      calls to ``handler``
+    :type intervals: :term:`iterable`
+    :param str plugin: optional job's plugin
+    :param str label: optional job's label
```

Comment:
```suggestion
    :param str plugin: optional plugin name to which the job belongs
    :param str label: optional label (name) for the job
```

Further suggestions invited! I'm just doing my best with the slightly odd (I assume French-inspired) thing you sometimes do with possessives.

Reply:
I need a (semi) advanced English grammar course. :-/